bitrise.yml

```diff
@@ -27,6 +27,21 @@ workflows:
     - run_e2e_tests
     - _notify_failure_on_slack
 
+  tagged_commit:
+    after_run:
+    - _make_dependencies
+    - run_linter
+    - run_unit_tests
+    - run_integration_tests
+    - check_dependency_managers
+    - ship
+    - _deploy_artifacts
```

Comment:
Does this workflow report to GH Checks somehow? I'm wondering how we can add observability to it, ensuring we never overlook its failure. Calling the `_notify_failure_on_slack` in this workflow can be an idea.